### PR TITLE
custom-set: add test for removing duplicates

### PR DESCRIPTION
**Pull request by senekor**

### CODE REVIEW

**Comment by senekor** [comment]
[forum post](https://forum.exercism.org/t/custom-set-new-test-against-deduplication/11130)

**Review by IsaacG — APPROVED** [review]

**Review by BNAndras — APPROVED** [review]

File: exercises/custom-set/canonical-data.json [hunk 1]
```diff
@@ -396,6 +396,16 @@
...
+        },
+        {
+          "uuid": "20d0a38f-7bb7-4c4a-ac15-90c7392ecf2b",
+          "description": "difference removes all duplicates in the first set",
+          "property": "difference",
+          "input": {
+            "set1": [1, 1],
+            "set2": [1]
+          },
+          "expected": [0]
```
Comment (glennj) [inline exercises/custom-set/canonical-data.json:408]: Why is the expected set `[0]`? Where does element `0` come from? Shouldn't I expect an empty set as the difference?

Comment (ErikSchierboom) [reply exercises/custom-set/canonical-data.json:408]: Yeah, I think that might be a mistake?

Comment (senekor) [reply exercises/custom-set/canonical-data.json]: Yes, that was a mistake. thanks!

**Review by ErikSchierboom — APPROVED** [review]
